packages/api/cms-api/schema.gql

```diff
@@ -403,6 +464,11 @@ type Mutation {
   updatePageTreeNodeCategory(id: ID!, category: String!): PageTreeNode!
   createPageTreeNode(input: PageTreeNodeCreateInput!, scope: PageTreeNodeScopeInput!, category: String!): PageTreeNode!
   currentUserSignOut: String!
+  userPermissionsCreatePermission(data: CreateUserPermissionInput!): UserPermission!
+  userPermissionsUpdatePermission(data: UpdateUserPermissionInput!): UserPermission!
```

Comment:
we name `data` usually `input`

Reply:
https://github.com/vivid-planet/comet/pull/1223/commits/3d41b7594dff6331ee7f9d4c410131e7953b5beb